programs/format/Format.cpp

```diff
@@ -66,6 +67,7 @@ int mainEntryClickHouseFormat(int argc, char ** argv)
 
     try
     {
+        boost::program_options::store(boost::program_options::parse_command_line(argc, argv, desc), options);
```

Comment:
This line is redundant, the same thing is done 10 lines higher. Need to remove it.

Reply:
done

Reply:
> This line is redundant, the same thing is done 10 lines higher. Need to remove it.

done